### PR TITLE
2020 Performance Chapter 

### PR DESCRIPTION
**Pull request by thefoxis**
## This PR includes

- the 2020 Performance chapter draft content
- charts and fallback imagery alongside with captions and descriptions
- necessary meta information for the chapter
- up-to-date contributors to the chapter

## Related issues
#905 

## Notes
@rviscomi there are a couple of `TODOs` relating linking chapters / sections of the Performance chapter, but otherwise I think I addressed everything. let me know if you need anything or stuff needs further editing! ⚡

### CODE REVIEW

File: src/content/en/2020/performance.md [hunk 1]
```diff
@@ -1,23 +1,338 @@
...
+
+Additionally, **the score is now calculated using different reference points for desktop and mobile**. What this means in practice is that it will be less forgiving on desktop (expecting fast websites) and more relaxed on mobile (since benchmark performance on mobile is less quick than desktop). You can compare your Lighthouse 5 and 6 score difference in [the Lighthouse Score calculator](https://googlechrome.github.io/lighthouse/scorecalc/). So, how did the scores really change?
+
+{{ figure_markup(
+  image="performance-change-in-lighthouse-score.png",
+  caption="Difference in Lighthouse Performance Score between versions 5 and 6.",
+  description="Column chart showcasing the change in Performance Score between versions 5 and 6. The highest percentage of websites (4%) observed no changes to the score, and the number of sites with score decreases outweighs score improvements.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vT3ukFCTxx__dTBLwDQ3K_hCtXdLRGOGUYdR_TO043n_aCTdcwkxPUku9Qfoal6BESiV5RMtd0nEbqT/pubchart?oid=786955541&format=interactive"
+  )
+}}
```
Comment (bazzadp) [inline src/content/en/2020/performance.md:50]: We need to add `sheets_gid` and `sql_file` parameters to all the figures:

```suggestion
{{ figure_markup(
  image="performance-change-in-lighthouse-score.png",
  caption="Difference in Lighthouse Performance Score between versions 5 and 6.",
  description="Column chart showcasing the change in Performance Score between versions 5 and 6. The highest percentage of websites (4%) observed no changes to the score, and the number of sites with score decreases outweighs score improvements.",
  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vT3ukFCTxx__dTBLwDQ3K_hCtXdLRGOGUYdR_TO043n_aCTdcwkxPUku9Qfoal6BESiV5RMtd0nEbqT/pubchart?oid=786955541&format=interactive",
  sheets_gid="518150031",
  sql_file="lh6_vs_lh5_performance_score_distribution.sql"
  )
}}
```

**Review by bazzadp — CHANGES_REQUESTED** [review]
This looks fantastic! I've not had a chance to read it all yet but from a brief skim through the markup this looks to be in the right format first time! Well done!

One thing missed though was new `sheets_gid` and `sql_file` parameters that we've added this year (so won't have been on last year's chapter if you used that as your base). We hope to expose the data more as detailed in #1570 to allow others to explore.

Would you or @max-ostapenko  or @dooman87 be able to add those?

Other than that think this is good to merge to move on to Editing stage.

File: src/content/en/2020/performance.md (hunk 1 above)
Comment (thefoxis) [reply src/content/en/2020/performance.md]: @bazzadp done 👍 hope I didn't mismatch anything there 😄

**Review by rviscomi — APPROVED** [review]
This looks great, excellent job @thefoxis!

**Pull request merged**